.github/CODEOWNERS

```
@@ -0,0 +1,48 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+# These owners will be the default owners for everything in the repo.
+# * <@insert_github_username>
+#
+# Order is important. The last matching pattern has the most precedence.
+
+
+# This file is also being managed automatically by the gitown tool.
+*                   @radxa-docs/Editor
+**/compute-module   @vamrs-feng @Ken-Vamrs
+**/cm3              @vamrs-feng
+**/cm3i             @vamrs-feng
+**/cm5              @Ken-Vamrs
+**/nx5              @vamrs-feng
+**/nio              @RadxaStephen
+**/rock3            @Radxa-Alvin @RadxaMitchell @Ken-Vamrs
+**/e25              @Radxa-Alvin
+**/rock3a           @RadxaMitchell
+**/rock3b           @Ken-Vamrs
+**/rock3c           @Radxa-Alvin
+**/rock4            @RadxaStephen
+**/rock5            @RadxaStephen @yjxyx # liuxian@radxa.com
+**/rock5a           @RadxaStephen
+**/rock5b           @RadxaStephen
+**/rock5itx         @yjxyx # liuxian@radxa.com
+**/rockpi           @RadxaMitchell @Ken-Vamrs
+**/rockpie          @RadxaMitchell
+**/rockpis          @RadxaMitchell
+**/rocks0           @Ken-Vamrs
+**/x
```

```suggestion
**/x                @shawnchainw
```
这样加上去后会不会在重新运行gitown时清除掉？

---

gitown只会在末尾添加单个文件的owner

---

文件开头的注释是它添加的么？如果可以最好在我们人工设置的规则和后面自动添加的规则中间加一个注释行分割一下

---

开头注释是gitown添加的，一开始的提交有用注释分类，但是都被gitown给格式化了